nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -112,6 +112,17 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor RIGHT_BOUND_WHERE = new PropertyDescriptor.Builder()
```

Comment:
Is there any situation where we'd want this false? I'm thinking this might be another case where changing the behavior is a fix and doesn't necessarily need to offer another property to maintain original behavior.